webapp/components/user_settings/user_settings_security/user_settings_security.jsx

```diff
@@ -944,6 +944,12 @@ export default class SecurityTab extends React.Component {
                     server_error={this.state.serverError}
                     updateSection={updateSectionStatus}
                     width='full'
+                    cancelButtonText={
+                        <FormattedMessage
+                            id='user.settings.security.close'
+                            defaultMessage='Close'
+                        />
+                    }
                 />
             );
         }
```

Comment:
Id for Close button evaluates to "[object_Object]Cancel" due to passed help text and div wrapper. Do we really need to pass it here?